src/site/static/eligibility.liquid

```diff
@@ -1974,26 +1974,39 @@ pageClass: "page-public-assistance"
   //   https://stgenssa.sccgov.org/debs/policy_handbook_GA/gachap07.pdf (7. Citizens/Noncitizens)
   //     Qualified non-citizen: Need to determine all non-citizen groups which fit this category. May or may not need to add more detail to immigration status form page.
   //
-  //   https://stgenssa.sccgov.org/debs/policy_handbook_Charts/ch-ga.pdf (Section 4.2)
-  //     Implement Grant level - UNSHARED Housing in Section 4.2.1. Also, max gross income cannot exceed max grant level, and max grant level changees with family size and living arrangement.
-  //
   //   https://stgenssa.sccgov.org/debs/policy_handbook_GA/gachap05.pdf (Section 5.1)
   //     Maximum age can be over 64 years with some conditions. May or may not need to implement this.
   function gaEligible() {
-    const age = document.getElementById("age").value || null;
+    const age = getValueOrNull('age');
     const numDependents = document.querySelectorAll('input[id^="hh-member-dependent"]:checked').length;
 
+     //   https://stgenssa.sccgov.org/debs/policy_handbook_Charts/ch-ga.pdf (Section 4.2)
+     //     UNSHARED Housing in Section 4.2.1. max gross income cannot exceed max grant level, and max grant level changees with family size and living arrangement.
+     //     In Section 4.2.1, the maximum grant amounts are listed on a monthly basis and has been converted to an yearly basis in the table below.
+    const grossLimit = MonthlyIncomeLimit.fromAnnual([
```

Comment:
You can create a `MonthlyIncomeLimit` directly so that we're inputting exactly the values in table 4.2.1 instead of first doing a `* 12` manual conversion:

```
const grossLimit = new MonthlyIncomeLimit([
  343,
  460,
  // etc
]);
```